server/src/helpers/formatAmount.js

```diff
@@ -0,0 +1,2 @@
+const formatAmount = value => Number(parseFloat(value).toFixed(2));
```

Comment:
Please add a JSDoc

Reply:
done...

Reply:
This should be `string` not `any` please update it in your next PR